core/server/api/utils.js

```diff
@@ -206,7 +207,7 @@ utils = {
          * @returns {Object} options
          */
         return function doHandlePermissions(options) {
-            var unsafeAttrObject = unsafeAttrNames && _.has(options, 'data.[' + docName + '][0]') ? _.pick(options.data[docName][0], unsafeAttrNames) : {},
+            let unsafeAttrObject = unsafeAttrNames && _.has(options, 'data.[' + docName + '][0]') ? _.pick(options.data[docName][0], unsafeAttrNames) : {},
```

Comment:
Wanne use string literals here? There are one two more places in the API folder.